### PR TITLE
Improve Advisor card loading skeleton

### PR DESCRIPTION
**Pull request by leSamo**
## Before
Screenshot from 2024-08-09 17-32-37

## After
Screenshot from 2024-08-09 17-31-58

## Loaded state
Screenshot from 2024-08-09 17-31-34

### CODE REVIEW

File: src/SmartComponents/Advisor/Advisor.js
Comment (gkarat) [inline src/SmartComponents/Advisor/Advisor.js:168]: Here we can't use optional chaining given that the value is present
```suggestion
                                                {intl.formatMessage(messages.incidents, { incidents: advisorIncidents.meta.count })}
```

**Review by gkarat — APPROVED** [review]
LGTM, just one minor note

**Comment by gkarat** [comment]
I have also restarted Travis test job. Let's wait for 🟢 before we can merge

**Comment by leSamo** [comment]
@gkarat the Travis job is broken, I will look into it in a separate PR

**Pull request merged**
